requirements.txt

```diff
@@ -5,3 +5,4 @@ requests
 schema-salad >= 8.8.20250205075315,<9
 ruamel.yaml >= 0.17.6, < 0.19
 typing_extensions;python_version<'3.10'
+cwlformat >= 2022.2.18
```

Comment:
No, this is brought in the the extra `pretty`; we don't want to require all users of `cwl-utils` to install `cwlformat`.
```suggestion
```

Reply:
Can you elaborate on this please? It's been a while since I actually put in the PR.

Why don't we want cwlformat as a requirement? 
What do you mean by the extra 'pretty'?

Reply:
@mr-c following this up, would really like to move away from the old ruamel.yaml load + dump formats since it is no longer supported in the 0.18 version of ruamel.yaml.